cmd/k3s/main.go

```diff
@@ -100,6 +102,23 @@ func findDataDir() string {
 	return dataDir
 }
 
+// findPreferBundledBin searches for prefer-bundled-bin from the config file, then CLI args.
+// we use pflag to process the args because we don't have a cli.Context yet
```

Comment:
Technically we do, `stageAndRun` is called from `stageAndRunCLI` which does have a `cli.Context`, its just that I believe this is the stub context that doesn't have all the various flags bound to it yet, at least I think it doesn't? I get lost in this bit.

Reply:
Updated comment